curvefs/src/client/dentry_cache_manager.cpp

```diff
@@ -63,10 +63,12 @@ CURVEFS_ERROR DentryCacheManagerImpl::GetDentry(uint64_t parent,
     return CURVEFS_ERROR::OK;
 }
 
-CURVEFS_ERROR DentryCacheManagerImpl::CreateDentry(const Dentry &dentry) {
+CURVEFS_ERROR DentryCacheManagerImpl::CreateDentry(const Dentry &dentry,
+                                                   const struct timespec &now) {
     std::string key = GetDentryCacheKey(dentry.parentinodeid(), dentry.name());
     NameLockGuard lock(nameLock_, key);
-    MetaStatusCode ret = metaClient_->CreateDentry(dentry);
+    MetaStatusCode ret = metaClient_->CreateDentry(dentry,
```

Comment:
You can get time here and no need to pass it from above.

Reply:
ok